src/main/scala/com/typesafe/sbt/SbtProguard.scala

```diff
@@ -86,21 +105,20 @@ object SbtProguard extends AutoPlugin {
       val filters = (filteredInputsValue flatMap (_.filter)).toSet
       val combinedFilter = if (filters.nonEmpty) Some(filters.mkString(",")) else None
       Seq(Filtered(mergeDirectoryValue, combinedFilter))
-    } else filteredInputsValue
+    }
   }
 
   lazy val proguardTask: Def.Initialize[Task[Seq[File]]] = Def.task {
-    writeConfiguration(proguardConfiguration.value, proguardOptions.value)
```

Comment:
I have a feeling that the configuration was written to a file to handle a lot of options, without hitting limits on the command line.

Reply:
Ah, maybe you're right. I switched to this because it was helpful for debugging to see the command being run

Reply:
update: I changed it back to writing the config to a file